backends/redis.go

```diff
@@ -0,0 +1,57 @@
+package backends
+
+import (
+	"context"
+	log "github.com/Sirupsen/logrus"
+	"github.com/go-redis/redis"
+	"github.com/prebid/prebid-cache/config"
+	"strconv"
+)
+
+type Redis struct {
+	cfg    config.Redis
+	client *redis.Client
+}
+
+func NewRedisBackend(cfg config.Redis) *Redis {
+	constr := cfg.Host + ":" + strconv.Itoa(cfg.Port)
+	client := redis.NewClient(&redis.Options{
+		Addr:     constr,
+		Password: cfg.Password,
+		DB:       cfg.Db,
+	})
+
+	_, err := client.Ping().Result()
+
+	if err != nil {
+		log.Fatalf("Error creating Redis backend: %v", err)
+		panic("Failed Connecting to Redis.")
```

Comment:
`log.Fatalf` runs `os.Exit`. You can remove the `panic` here... it should never happen.